doc/reference/configuration/cfg_logging.rst

```diff
@@ -42,36 +46,41 @@
 
     Since version 1.7.4.
     By default, Tarantool sends the log to the standard error stream
-    (``stderr``). If ``log`` is specified, Tarantool sends the log to a file,
-    or to a pipe, or to the system logger.
+    (``stderr``). If ``log`` is specified, Tarantool can send the log to a ...
+
+    * file;
+
+    * pipe;
+
+    * system logger.
 
-    Example setting for sending the log to a file:
+    The example below shows how to send logs to the ``tarantool.log`` file:
```

Comment:
Idea: make examples less "narrative" but more structured: 

```suggestion
    Example 1: sending the log to the ``tarantool.log`` file.
```

Reply:
Yep, I like it.